### PR TITLE
Prefer redirecting to only the latest (stable) version of a pod

### PR DESCRIPTION
**Pull request by segiddins**
\c @orta @floere

### CODE REVIEW

**Comment by orta** [comment]
Yeah, this looks :+1: to me

File: classes/spec_metadata_generator.rb [hunk 1]
```diff
@@ -24,6 +24,14 @@ def save
     File.open(json_filepath, "wb") { |f| f.write hash_string }
   end
 
+  def latest_version
+    versions.reverse_each.find { |v| !v.prerelease? } || versions.last
```
Comment (floere) [inline classes/spec_metadata_generator.rb]: I assume the versions are correctly sorted? (I can see the code on line 13, but am not 100% sure I get that guarantee)

Comment (segiddins) [inline classes/spec_metadata_generator.rb]: correct. they're sorted in `generate`

**Comment by floere** [comment]
One thing – calculating the latest version is done multiple times in a row (afaics, 3x). Would memoizing be a good idea?

**Comment by floere** [comment]
Other than that I am happy :+1:

**Comment by orta** [comment]
Cool, I'm chill about the memoizing stuff.

**Pull request merged**